README.md

```diff
@@ -48,6 +48,7 @@
 - [x] Validations and Model specs
 - [x] Setup and controllers
 - [x] Controllers specs
+- [x] Views
 
 <p align='right'>(<a href='#readme-top'>back to top</a>)</p>
```

Comment:
| Descriptive Pull Request Summary 🟢 📝 |
|-|
| ![image](https://github.com/Atril33/rails_blog_app/assets/93039359/aac9fe05-70fd-4aee-a4b9-62196fd31ad7) |

**[OPTIONAL]**

- [ ] It would be nice if you made your pull request summary more detailed to make your pull request more descriptive and professional 😊

Reply:
Thanks @HAMMAS-SALEEM, I'll make sure to make well detailed pull request from next time.